packages/languages/src/node/language-server-contribution.spec.ts

```diff
@@ -0,0 +1,41 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { ProcessErrorEvent } from '@theia/process/lib/node';
+import { BaseLanguageServerContribution } from './language-server-contribution';
+
+/**
+ * Typescript compiler should try to compile these files,
+ * and it should fail if the types are not compatible.
+ */
+
+// @ts-ignore
+class Extension1 extends BaseLanguageServerContribution {
+
+    protected onDidFailSpawnProcess(error: Error) {
```

Comment:
When I switched to master (keeping this file), it failed at build time with what looks like the same error as in the issues. When I build from this PR's branch, no more errors.